### PR TITLE
Clean & Minimal Homepage :sparkles:

### LINKED ISSUE #3582: update homepage
You can find the mockup [here](https://www.figma.com/file/BwA7cNkepuEqD9b39u3VQK/changes-v1?node-id=1%3A2513&t=QMGz197OiFGafWk6-1)

Note: we left all resolved comments open to making it easier for you to see what has changed. Feel free to resolve them once you have implemented them

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
fixes #3582

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3589"></a>

### CODE REVIEW

File: src/lib/components/forms/newsletter.svelte [hunk 1]
```diff
@@ -9,9 +9,10 @@
   let resultMessage = "";
   let isSubmissionInProgress = false;
 
-  export let title: string = "Read our newsletter";
+  export let title: string =
+    "Read about best practices in developer experience";
   export let description: string =
-    "Access Gitpod's latest thinking, news, and insights.";
+    "Read about best practices in developer experience";
```
Comment (Siddhant-K-code) [inline src/lib/components/forms/newsletter.svelte:15]: @taliamoyal Need your help here for copyText

**Comment by Siddhant-K-code** [comment]
Left this component untouched. Because,  I think, We should keep the examples to ASAP launch a workspace.
image

File: src/lib/components/forms/newsletter.svelte (hunk 1 above)
Comment (taliamoyal) [reply src/lib/components/forms/newsletter.svelte:15]: responded in figma!

File: src/routes/index.svelte [hunk 2]
```diff
@@ -53,22 +46,17 @@
...
+<Features {features} />
 <IntegrateDontDictate class="py-small" />
 <Features features={otherFeatures} />
-<Section>
-  <HomepageResources {cards} />
-</Section>
 <Testimonials
-  title="Trusted by the best developer teams"
-  text="Gitpod empowers teams to be always ready to code."
+  title="Accelerate your software development."
```
Comment (chrifro) [inline src/routes/index.svelte:57]: Could you add 48px spacing instead of 16px between the headline and the quotes?

**Review by chrifro — COMMENTED** [review]
Nice thanks @Siddhant-K-code ✨

Few comments:
- could you keep these cards consistent with a 320px width? That should work fine with the current copy. Otherwise it looks unbalanced and it breaks too early on smaller screen sizes
Bildschirm­foto 2023-04-20 um 11 59 47
- could you add the boxes to these icons? (see [Figma](https://www.figma.com/file/BwA7cNkepuEqD9b39u3VQK/changes-v1?node-id=1%3A2513&t=frbABXiiglQx4blo-1))
Bildschirm­foto 2023-04-20 um 12 02 43

- align this section for smaller screen sizes (left align, consistent spacing between headlines and text (16px), and buttons (16px) instead of 32px
Bildschirm­foto 2023-04-20 um 12 08 43

**Comment by Siddhant-K-code** [comment]
> could you keep these cards consistent with a 320px width? That should work fine with the current copy. Otherwise it looks unbalanced and it breaks too early on smaller screen sizes


But, then it would look weird on regular screen sizes:

image

**Comment by Siddhant-K-code** [comment]
> could you add the boxes to these icons? (see [Figma](https://www.figma.com/file/BwA7cNkepuEqD9b39u3VQK/changes-v1?node-id=1%3A2513&t=frbABXiiglQx4blo-1))

It's already there, try right-clicking and open image in new tab to verify. I think, it is getting camouflaged due to background color of that section

**Comment by chrifro** [comment]
>But, then it would look weird on regular screen sizes:

What if we then minimally adjust all card's widths? What width is needed to make it fit? e.g. we could adjust them all to 328px instead of 320px.

**Comment by Siddhant-K-code** [comment]
> But, then it would look weird on regular screen sizes:
> 
> What if we then minimally adjust all card's widths? What width is needed to make it fit? e.g. we could adjust them all to 328px instead of 320px.

Done, It settled for 344px 😅

**Pull request merged**